package.json

```diff
@@ -39,6 +39,7 @@
     "@types/testing-library__jest-dom": "^5.9.5",
     "@typescript-eslint/eslint-plugin": "^4.20.0",
     "@typescript-eslint/parser": "^4.20.0",
+    "@uireact/button": "file:packages/button",
```

Comment:
This is so I can use the button inside the uireact package, as an example for instance.